detector/include/FieldMapXYZ.h

```diff
@@ -17,8 +17,8 @@ class FieldMapXYZ: public DD4hep::Geometry::CartesianField::Object {
       Bx(_Bx), By(_By), Bz(_Bz) {}
   };
 
-  int nX, nY, nZ;
-  
+  std::string CoorsOrder;
```

Comment:
CoorOrder is a string that tells you what the coordinates ordering in the tree. For example, in the case of FieldMapBrBz, it can have the values "RZ" or "ZR". In the first case, the coordinates are scanned first R and then Z in the tree. In the second case the scanning is on the other way around.

I changed this following a previous comment from you. Now CoorOrder is an integer which takes the values 1 or 2 depending if the ordering is "RZ" or "ZR", respectively.